### PR TITLE
677 add icon url in survey dashboard

### LINKED ISSUE #677: Add icon_url in survey_dashboard
Add in survey dashboard the icon_url field created for survey.

### PR DESCRIPTION
**Pull request by philipeleandro**
fix #677 

@PolinaStamenova @edimossilva

### CODE REVIEW

**Comment by PolinaStamenova** [comment]
@philipeleandro 👍

**Comment by philipeleandro** [comment]
@PolinaStamenova ok, I'll use these commands.

File: backend/spec/system/admin/survey_crud_spec.rb [hunk 1]
```diff
@@ -117,6 +131,14 @@
         it 'does not see surveys that not belongs to him' do
           expect(page).not_to have_text(survey2.name)
         end
+
+        it 'see icon_url field that belongs to him' do
+          expect(page).to have_text(survey1.icon_url)
+        end
+
+        it 'does not icon_url field that not belongs to him' do
+          expect(page).not_to have_text(survey2.icon_url)
```
Comment (PolinaStamenova) [inline backend/spec/system/admin/survey_crud_spec.rb:140]: Maybe you can add the verb `see` to the description of the test: 
```
it 'does not see icon_url field that not belongs to him' do
```

**Review by PolinaStamenova — CHANGES_REQUESTED** [review]
LGFM 👍

File: backend/spec/system/admin/survey_crud_spec.rb (hunk 1 above)
Comment (philipeleandro) [reply backend/spec/system/admin/survey_crud_spec.rb:140]: Thanks for the review @PolinaStamenova, I'll add it.

File: backend/spec/system/admin/survey_crud_spec.rb [hunk 2]
```diff
@@ -117,6 +131,14 @@
         it 'does not see surveys that not belongs to him' do
           expect(page).not_to have_text(survey2.name)
         end
+
+        it 'see icon_url field that belongs to him' do
```
Comment (edimossilva) [inline backend/spec/system/admin/survey_crud_spec.rb:135]: sees

**Review by edimossilva — APPROVED** [review]

**Pull request merged**